packages/blockchain-wallet-v4-frontend/src/scenes/Login/components/UrlNoticeBar/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import { Icon, Link, Text } from 'blockchain-info-components'
+import { media } from 'services/styles'
+
+const BACKGROUND_COLOR = '#0C6CF2'
```

Comment:
this should be using colors from the theme, not hardcoded

Reply:
Done!